socorro/external/es/crashstorage.py

```diff
@@ -655,3 +656,18 @@ def _submit_crash_to_elasticsearch(
                     exc_info=True,
                 )
                 raise
+
+    def delete_crash(self, crash_id):
```

Comment:
`socorro/ext/crashstorage.py:CrashStorageBase` has this as `remove`. I think I like `delete_crash` better since it lines up with the other methods (`save_raw_crash`, `get_processed_crash`, etc) and is clearer about what it's doing.

Can you rename the `remove` stub in `CrashStorageBase` to `delete_crash`? You'll need to update all the `CrashStorageBase` subclasses that implemented `remove`, too.

```
> rg "def remove"
crashstorage_base.py
198:    def remove(self, crash_id):
260:    def remove(self, crash_id):

fs/crashstorage.py
234:    def remove(self, crash_id):
```